packages/rocketchat-i18n/i18n/cs.i18n.json

```diff
@@ -1133,6 +1133,8 @@
   "Offline_form": "Offline formulář",
   "Offline_form_unavailable_message": "Zpráva nedostupného offline formuláře",
   "Offline_Link_Message": "PŘEJÍT NA ZPRÁVU",
+  "Offline_Mention_Email": "Zmínka od __user__ v #__room__",
```

Comment:
We have to remove these two lines. `Offline_Mention_Email` & `Offline_Mention_All_Email`. These has been changed to https://github.com/RocketChat/Rocket.Chat/pull/7342/files#diff-900a71156bf66b90776f869f3f63ccd9R1191 as subjects are moved to settings.